### PR TITLE
Event addon migration

### PR DESCRIPTION
**Pull request by paulius-sladkevicius**
Related #20

### CODE REVIEW

**Comment by bwrsandman** [comment]
Forgot to say in #20:
Just update `openerp/openupgrade/doc/source/modules70-80.rst` to reflect the coverage of the module.
You will need to rebase with upstream/master

```
git remote add upstream https://github.com/OpenUpgrade/OpenUpgrade.git
git pull --rebase upstream master
```

**Comment by paulius-sladkevicius** [comment]
@bwrsandman I've added it

**Comment by bwrsandman** [comment]
@komsas Thank you!
:+1:

File: addons/event/migrations/8.0.0.1/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---Fields in module 'event'---
+event        / event.event              / date_tz (selection)           : NEW selection_keys: function
+event        / event.event              / description (html)            : NEW
+event        / event.event              / main_speaker_id (many2one)    : DEL relation: res.partner
+event        / event.event              / message_last_post (datetime)  : NEW
+# Used set_message_last_post()
+
+event        / event.event              / note (text)                   : DEL
+# Fields argument 'oldname' used, now points to 'description' field
+
```
Comment (StefanRijnhart) [inline addons/event/migrations/8.0.0.1/openupgrade_analysis_work.txt:12]: Please convert to html using openerp.tools.mail.plaintext2html

Comment (paulius-sladkevicius) [inline addons/event/migrations/8.0.0.1/openupgrade_analysis_work.txt:12]: @StefanRijnhart  Could you explain why it should be done?
If this manner everybody should do that? e.g. in the #9

Comment (paulius-sladkevicius) [inline addons/event/migrations/8.0.0.1/openupgrade_analysis_work.txt:12]: Please ignore last questions.

**Comment by paulius-sladkevicius** [comment]
Added generic function that can convert text value to HTML.

**Comment by bwrsandman** [comment]
@komsas nice function, but wouldn't it be more useful to put it in `openerp.openupgrade.openupgrade80`?

**Comment by paulius-sladkevicius** [comment]
Firstly we should decide if it's worth to put there. Needs to check if there more places where it could be used.

**Comment by bwrsandman** [comment]
@StefanRijnhart @hbrunn 

> wouldn't it be more useful to put `convert_field_to_html()` in openerp.openupgrade.openupgrade80?

**Comment by legalsylvain** [comment]
Agree with @bwrsandman remarks.

**Comment by StefanRijnhart** [comment]
As it is already  refactored out, we might as well have it in the API. I would make that openupgrade.py (not openupgrade_80.py) because this is not particularly 8.0 specific.

**Comment by paulius-sladkevicius** [comment]
@StefanRijnhart in such case I'll try to move it to API

**Comment by paulius-sladkevicius** [comment]
Sorry, looks by mistake I've merged 8.0 while resolving conflict, but this pull request points to master. I'll prepare a new clean pull request to 8.0.

**Pull request closed without merging**